cost/aws/old_snapshots/aws_delete_old_snapshots.pt

```diff
@@ -655,7 +644,7 @@ EOS
       field "savings" do
         label "Estimated Monthly Savings"
       end
```

Comment:
For these five AWS usage recommendation policies, could we further refine the "savings" field content?

In the incident violation_data, I noticed the `savings` value included currency info as well as an amount: [sample_responses/org_27018/incident_aws_unused_volumes.json#L310](https://github.com/flexera/optima-recommendations/blob/3da2ff02a004487a50396f2b6aa4401b7358350c/front_service/services/incident/testing/sample_responses/org_27018/incident_aws_unused_volumes.json#L310)
```
"savings": "US$ 0.83",
```
Could we instead maybe separate that, so it would be:
```
"savings": 0.83,
"savings_currency": "US$",
```
That way, we can scrape the "savings" amount for recommendations in a standard way, without any string processing.

Reply:
While I think making this change is a good idea, we should follow semantic versioning and bump the major version instead of the minor version since it could be considered a backward incompatible change. I believe this would probably result in most of the policies becoming `3.0` versions.

Reply:
Do we even need the currency anymore or can recommendations just pull it directly?

Reply:
I think it's great if the policy annotates the savings number with whatever currency information it happened to use.

Today, Optima just has a [currency_setting](https://reference.rightscale.com/bill_analysis/#/currency_setting/currency_setting_show) that is set for the entire org. 
But to really make sense of multiple currencies in the same org, we'll need to ensure all of our cost sources indicate what currency unit they're using, so we can avoid summing up costs with disparate currencies.

Reply:
yeah, but the policy just pulls in currency_setting to display it back to you: https://github.com/flexera-public/policy_templates/blob/57b01c17f8d486e3579041c646b0b34f6dd1d68c/cost/aws/idle_compute_instances/idle_compute_instances.pt#L224

Reply:
Good point! 😄 
I guess that currency info could still be useful to those viewing the raw incident?